payments/models.py

```diff
@@ -296,6 +298,74 @@ def name(self):
     )
 
 
+class TalpaProductAccountingManager(models.Manager):
+    def get_product_accounting_for_order(self, order):
+        """
+        Talpa ecom requirements for accounting of products defined in Talpa:
+        * Orders in the EAST/WEST regions need to be for separate products
+        * Orders for winter storage (including storage on ice) and berths need to be separate products
+        """
+
+        if order.order_type == OrderType.ADDITIONAL_PRODUCT_ORDER:
+            if (
+                order.order_lines.count() == 1
+                and order.order_lines.all().first().product.service
+                == ProductServiceType.STORAGE_ON_ICE
+            ):
+                product_type = TalpaProductType.WINTER
+                region = order.lease.berth.pier.harbor.region
```

Comment:
Storage on ice woks only for berth leases, right? Should we somehow catch the case where the order doesn't have a lease?